rust-runtime/aws-smithy-http-server-python/README.md

```diff
@@ -2,6 +2,62 @@
 
 Server libraries for smithy-rs generated servers, targeting pure Python business logic.
 
+## Running servers on AWS Lambda
+
+`aws-smithy-http-server-python` supports running your services on [AWS Lambda](https://aws.amazon.com/lambda/).
+
+You need to use `run_lambda` method instead of `run` method to start
+the [custom runtime](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-custom.html)
+instead of the [Hyper](https://hyper.rs/) HTTP server.
+
+In your `app.py`:
+
+```diff
+from libpokemon_service_server_sdk import App
+from libpokemon_service_server_sdk.error import ResourceNotFoundException
+
+# ...
+
+# Get the number of requests served by this server.
+@app.get_server_statistics
+def get_server_statistics(
+    _: GetServerStatisticsInput, context: Context
+) -> GetServerStatisticsOutput:
+    calls_count = context.get_calls_count()
+    logging.debug("The service handled %d requests", calls_count)
+    return GetServerStatisticsOutput(calls_count=calls_count)
+
+# ...
+
+-app.run()
++app.run_lambda()
```

Comment:
What do you think about renaming the entrypoint of the standard server from `run()` to `run_server()` ?

Reply:
I think `run()` is clearer. I don't know if it is idiomatic but, if we decide to unify `run_lambda` and any possible `run_*` versions into a big `run` function with keyword arguments like `run(lambda=True, foo="bar", ...)` that would allow us to do without renaming it again.

Reply:
Ok, makes sense!